### ISSUE TITLE
Some errors

### ISSUE THREAD

**Issue by nzmedet** [gkoberger/startupnotes#4]
First error `nokogiri 1.6.0` cannot be installed<br>
when using version higher, second error: `libv8` cannot be installed<br>
when installing `libv8 --with-system-v8` another error `rubytracer` cannot be installed<br>
when uninstalling `libv8` and installing `rubytracer` another error `libv8` cannot be installed<br>
then i gave up,<br>
is there any online version?

**Comment by pvorb** [comment]
@vko-online Yes, there's an online version at [startupnotes.org](http://startupnotes.org/).
